HHS/Head-Start-TTADP
# [TTAHUB-799] Upgrade Font Awesome to Version 6 Pro

## Description of change

Upgrade Font Awesome to Version 6 Pro

I have also installed the optional versions of the icons (solid, outline, etc) and kept the free versions.

## How to test

Visit any pages in the app that use Font Awesome Icon. These icons should show as they did before.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-799


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
